pkgs/applications/networking/instant-messengers/profanity/default.nix

```diff
@@ -21,13 +25,25 @@ stdenv.mkDerivation rec {
     sha256 = "1f7ylw3mhhnii52mmk40hyc4kqhpvjdr3hmsplzkdhsfww9kflg3";
   };
 
+  enableParallelBuilding = true;
+
   nativeBuildInputs = [ pkgconfig ];
+
   buildInputs = [
     readline libuuid libmesode
     glib openssl expat ncurses libotr curl
   ] ++ optionals autoAwaySupport [ libXScrnSaver libX11 ]
     ++ optionals notifySupport   [ libnotify gdk_pixbuf ]
-    ++ optionals traySupport     [ gnome2.gtk ];
+    ++ optionals traySupport     [ gnome2.gtk ]
+    ++ optionals gpgSupport      [ gpgme ]
+    ++ optionals pythonSupport   [ python ];
```

Comment:
What python version is preferred here? Archlinux builds against python3

Reply:
This is set at https://github.com/Moredread/nixpkgs/blob/ebd3a4622453665df5d600ab4a69c9d23060b88c/pkgs/top-level/all-packages.nix#L16067-L16068 which is python 2. I intended to use python 3, I'll change it.